edgexfoundry/app-functions-sdk-go
# fix: Use released SDK version in App Template

fixes #1427







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **N/A existing suffice**
      <link to docs PR>

## Testing Instructions
Temporary comment out the replace statement in the go.mod file
run make build from app-service-template folder
verify service builds 
run make test from app-service-template folder
verify all tests pass

## New Dependency Instructions (If applicable)
